src/client/components/configuration/style.css

```diff
@@ -0,0 +1,48 @@
+.configuration {
+	flex-grow: 1;
+	flex-shrink: 1;
+	display: flex;
+	flex-direction: column;
+    font-weight: normal;

```

Comment:
Probably should look into adding something like https://github.com/stylelint/stylelint